exchanges/kucoin/kucoin_test.go

```diff
@@ -1347,624 +1396,640 @@ func TestPostFuturesOrder(t *testing.T) {
 	t.Parallel()
-	if _, err := ku.FetchOrderbook(context.Background(), spotTradablePair, asset.Spot); err != nil {
-		t.Error(err)
-	}
-	if _, err := ku.FetchOrderbook(context.Background(), marginTradablePair, asset.Margin); err != nil {
-		t.Error(err)
-	}
-	if _, err := ku.FetchOrderbook(context.Background(), futuresTradablePair, asset.Futures); err != nil {
-		t.Error(err)
+	var result *orderbook.Base
+	var err error
+	for assetType, tp := range assertToTradablePairMap {
+		result, err = ku.FetchOrderbook(context.Background(), tp, assetType)
+		require.NoErrorf(t, err, "%w asset type: %s pair: %s", assetType.String(), tp.String())
+		assert.NotNilf(t, result, "%w asset type: %s pair: %s", assetType.String(), tp.String())
 	}
 }
 
 func TestGetHistoricCandles(t *testing.T) {
 	startTime := time.Now().Add(-time.Hour * 48)
 	endTime := time.Now().Add(-time.Hour * 3)
+	var result *kline.Item
 	var err error
-	_, err = ku.GetHistoricCandles(context.Background(), futuresTradablePair, asset.Futures, kline.OneHour, startTime, endTime)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = ku.GetHistoricCandles(context.Background(), spotTradablePair, asset.Spot, kline.OneHour, startTime, time.Now())
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = ku.GetHistoricCandles(context.Background(), marginTradablePair, asset.Margin, kline.OneHour, startTime, time.Now())
-	if err != nil {
-		t.Fatal(err)
+	for assetType, tp := range assertToTradablePairMap {
+		result, err = ku.GetHistoricCandles(context.Background(), tp, assetType, kline.OneHour, startTime, endTime)
+		require.NoErrorf(t, err, "%w asset type: %s pair: %s", assetType.String(), tp.String())
+		assert.NotNilf(t, result, "%w asset type: %s pair: %s", assetType.String(), tp.String())
 	}
 }
 
 func TestGetHistoricCandlesExtended(t *testing.T) {
 	startTime := time.Now().Add(-time.Hour * 48)
 	endTime := time.Now().Add(-time.Hour * 1)
+	var result *kline.Item
 	var err error
-	_, err = ku.GetHistoricCandlesExtended(context.Background(), spotTradablePair, asset.Spot, kline.OneHour, startTime, endTime)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = ku.GetHistoricCandlesExtended(context.Background(), spotTradablePair, asset.Spot, kline.FiveMin, startTime, endTime)
-	if err != nil {
-		t.Error(err)
-	}
-	_, err = ku.GetHistoricCandlesExtended(context.Background(), marginTradablePair, asset.Margin, kline.OneHour, startTime, endTime)
-	if err != nil {
-		t.Fatal(err)
-	}
-	_, err = ku.GetHistoricCandlesExtended(context.Background(), futuresTradablePair, asset.Futures, kline.FiveMin, startTime, endTime)
-	if err != nil {
-		t.Error(err)
+	for assetType, tp := range assertToTradablePairMap {
+		result, err = ku.GetHistoricCandlesExtended(context.Background(), tp, assetType, kline.FiveMin, startTime, endTime)
+		require.NoErrorf(t, err, "%w asset type: %s pair: %s", assetType.String(), tp.String())
+		assert.NotNilf(t, result, "%w asset type: %s pair: %s", assetType.String(), tp.String())
```

Comment:
Could you please look into the asset.Futures response please?
I receive this where I add the following code
```go
   t.Logf("%+v\n", result.Candles)
```

```
Kucoin - [missing data between 2024-08-05 01:45:00 and 2024-08-05 02:15:00 missing data between 2024-08-05 02:25:00 and 2024-08-05 02:35:00 missing data between 2024-08-05 02:40:00 and 2024-08-05 03:15:00 missing data between 2024-08-05 03:30:00 and 2024-08-05 03:35:00 missing data between 2024-08-05 03:45:00 and 2024-08-05 04:25:00 missing data between 2024-08-05 04:30:00 and 2024-08-05 04:40:00 missing data between 2024-08-05 04:50:00 and 2024-08-05 05:00:00 missing data between 2024-08-05 05:10:00 and 2024-08-05 05:15:00 missing data between 2024-08-05 05:20:00 and 2024-08-05 05:45:00 missing data between 2024-08-05 05:50:00 and 2024-08-05 06:00:00 missing data between 2024-08-05 06:05:00 and 2024-08-05 06:20:00 missing data between 2024-08-05 06:30:00 and 2024-08-05 06:40:00 missing data between 2024-08-05 06:45:00 and 2024-08-05 07:15:00 missing data between 2024-08-05 07:20:00 and 2024-08-05 07:25:00 missing data between 2024-08-05 07:40:00 and 2024-08-05 07:45:00 missing data between 2024-08-05 07:50:00 and 2024-08-05 08:10:00 missing data between 2024-08-05 08:15:00 and 2024-08-05 08:25:00 missing data between 2024-08-05 08:30:00 and 2024-08-05 09:45:00 missing data between 2024-08-05 09:50:00 and 2024-08-05 10:15:00 missing data between 2024-08-05 10:20:00 and 2024-08-05 10:55:00 missing data between 2024-08-05 11:00:00 and 2024-08-05 11:25:00 missing data between 2024-08-05 11:30:00 and 2024-08-05 11:45:00 missing data between 2024-08-05 11:50:00 and 2024-08-05 12:55:00 missing data between 2024-08-05 13:40:00 and 2024-08-05 14:10:00 missing data between 2024-08-05 14:20:00 and 2024-08-05 14:25:00 missing data between 2024-08-05 14:30:00 and 2024-08-05 14:40:00 missing data between 2024-08-05 14:50:00 and 2024-08-05 15:00:00 missing data between 2024-08-05 15:05:00 and 2024-08-05 16:25:00 missing data between 2024-08-05 16:40:00 and 2024-08-05 17:30:00 missing data between 2024-08-05 17:35:00 and 2024-08-05 17:50:00 missing data between 2024-08-05 17:55:00 and 2024-08-05 18:40:00 missing data between 2024-08-05 18:45:00 and 2024-08-05 18:55:00 missing data between 2024-08-05 19:00:00 and 2024-08-05 19:15:00 missing data between 2024-08-05 19:20:00 and 2024-08-05 19:50:00 missing data between 2024-08-05 19:55:00 and 2024-08-05 21:45:00 missing data between 2024-08-05 21:50:00 and 2024-08-06 00:00:00 missing data between 2024-08-06 00:05:00 and 2024-08-06 00:25:00 missing data between 2024-08-06 00:30:00 and 2024-08-06 00:45:00 missing data between 2024-08-06 00:50:00 and 2024-08-06 01:00:00 missing data between 2024-08-06 01:05:00 and 2024-08-06 02:20:00 missing data between 2024-08-06 02:25:00 and 2024-08-06 03:25:00 missing data between 2024-08-06 03:30:00 and 2024-08-06 03:55:00 missing data between 2024-08-06 04:00:00 and 2024-08-06 06:30:00 missing data between 2024-08-06 06:35:00 and 2024-08-06 06:45:00 missing data between 2024-08-06 06:50:00 and 2024-08-06 07:20:00 missing data between 2024-08-06 07:25:00 and 2024-08-06 08:35:00 missing data between 2024-08-06 08:45:00 and 2024-08-06 08:55:00 missing data between 2024-08-06 09:00:00 and 2024-08-06 11:25:00 missing data between 2024-08-06 11:30:00 and 2024-08-06 12:15:00 missing data between 2024-08-06 12:20:00 and 2024-08-06 12:45:00 missing data between 2024-08-06 12:50:00 and 2024-08-06 12:55:00 missing data between 2024-08-06 13:00:00 and 2024-08-06 13:05:00 missing data between 2024-08-06 13:10:00 and 2024-08-06 13:25:00 missing data between 2024-08-06 13:30:00 and 2024-08-06 13:40:00 missing data between 2024-08-06 13:45:00 and 2024-08-06 14:10:00 missing data between 2024-08-06 14:20:00 and 2024-08-06 14:25:00 missing data between 2024-08-06 14:30:00 and 2024-08-06 14:35:00 missing data between 2024-08-06 14:50:00 and 2024-08-06 15:40:00 missing data between 2024-08-06 15:50:00 and 2024-08-06 16:05:00 missing data between 2024-08-06 16:15:00 and 2024-08-06 16:55:00 missing data between 2024-08-06 17:00:00 and 2024-08-06 17:30:00 missing data between 2024-08-06 17:35:00 and 2024-08-06 17:45:00 missing data between 2024-08-06 17:50:00 and 2024-08-06 18:05:00 missing data between 2024-08-06 18:10:00 and 2024-08-06 19:00:00 missing data between 2024-08-06 19:05:00 and 2024-08-06 19:40:00 missing data between 2024-08-06 19:45:00 and 2024-08-06 20:00:00 missing data between 2024-08-06 20:05:00 and 2024-08-06 20:15:00 missing data between 2024-08-06 20:20:00 and 2024-08-06 22:25:00 missing data between 2024-08-06 22:35:00 and 2024-08-06 22:40:00 missing data between 2024-08-06 22:45:00 and 2024-08-06 22:50:00 missing data between 2024-08-06 22:55:00 and 2024-08-06 23:20:00 missing data between 2024-08-06 23:25:00 and 2024-08-07 00:15:00 missing data between 2024-08-07 00:20:00 and 2024-08-07 00:40:00 missing data between 2024-08-07 00:45:00 and 2024-08-07 00:50:00]
    kucoin_test.go:1882: [{Time:2024-08-05 01:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 01:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 01:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:15:00 +1000 AEST Open:2230.8 High:2258.08 Low:2222.55 Close:2245.21 Volume:785 ValidationIssues:} {Time:2024-08-05 12:20:00 +1000 AEST Open:2246.2 High:2246.64 Low:2246.2 Close:2246.64 Volume:887 ValidationIssues:} {Time:2024-08-05 02:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:35:00 +1000 AEST Open:2275.57 High:2275.57 Low:2275.57 Close:2275.57 Volume:14 ValidationIssues:} {Time:2024-08-05 02:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 02:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 03:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 03:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 03:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 13:15:00 +1000 AEST Open:2296.36 High:2296.36 Low:2296.36 Close:2296.36 Volume:10 ValidationIssues:} {Time:2024-08-05 13:20:00 +1000 AEST Open:2321.65 High:2322.2 Low:2321.65 Close:2322.2 Volume:1000 ValidationIssues:} {Time:2024-08-05 13:25:00 +1000 AEST Open:2324.53 High:2327.4 Low:2321.76 Close:2325.6 Volume:8000 ValidationIssues:} {Time:2024-08-05 03:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 13:35:00 +1000 AEST Open:2311.31 High:2316.37 Low:2305.8 Close:2316.37 Volume:9001 ValidationIssues:} {Time:2024-08-05 13:40:00 +1000 AEST Open:2320.31 High:2320.31 Low:2320.31 Close:2320.31 Volume:50 ValidationIssues:} {Time:2024-08-05 03:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 03:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 03:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:25:00 +1000 AEST Open:2373.01 High:2373.01 Low:2369.17 Close:2369.17 Volume:94 ValidationIssues:} {Time:2024-08-05 04:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:40:00 +1000 AEST Open:2350.08 High:2359.8 Low:2349.01 Close:2359.8 Volume:14078 ValidationIssues:} {Time:2024-08-05 14:45:00 +1000 AEST Open:2347.31 High:2347.31 Low:2347.2 Close:2347.2 Volume:700 ValidationIssues:} {Time:2024-08-05 04:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 04:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:00:00 +1000 AEST Open:2329.31 High:2329.31 Low:2299.31 Close:2299.31 Volume:1140 ValidationIssues:} {Time:2024-08-05 15:05:00 +1000 AEST Open:2309.17 High:2309.17 Low:2309.17 Close:2309.17 Volume:30 ValidationIssues:} {Time:2024-08-05 05:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:15:00 +1000 AEST Open:2302.31 High:2302.31 Low:2302.31 Close:2302.31 Volume:295 ValidationIssues:} {Time:2024-08-05 05:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 05:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 05:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 05:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 05:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:45:00 +1000 AEST Open:2348.51 High:2348.51 Low:2340.85 Close:2340.96 Volume:241 ValidationIssues:} {Time:2024-08-05 05:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 05:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:00:00 +1000 AEST Open:2288.66 High:2288.66 Low:2286.35 Close:2286.35 Volume:211 ValidationIssues:} {Time:2024-08-05 06:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 06:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 06:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:20:00 +1000 AEST Open:2254.01 High:2254.01 Low:2164.81 Close:2164.81 Volume:971 ValidationIssues:} {Time:2024-08-05 16:25:00 +1000 AEST Open:2199.4 High:2208.49 Low:2179.32 Close:2208.49 Volume:3 ValidationIssues:} {Time:2024-08-05 06:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 06:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:40:00 +1000 AEST Open:2187.12 High:2193.9 Low:2187.12 Close:2193.9 Volume:2 ValidationIssues:} {Time:2024-08-05 06:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 06:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 06:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 07:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 07:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 07:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:15:00 +1000 AEST Open:2294.6 High:2295.04 Low:2294.6 Close:2295.04 Volume:560 ValidationIssues:} {Time:2024-08-05 07:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:25:00 +1000 AEST Open:2327.89 High:2330.77 Low:2327.89 Close:2330.77 Volume:73 ValidationIssues:} {Time:2024-08-05 17:30:00 +1000 AEST Open:2337.97 High:2337.97 Low:2337.97 Close:2337.97 Volume:1 ValidationIssues:} {Time:2024-08-05 17:35:00 +1000 AEST Open:2345.17 High:2345.28 Low:2341.91 Close:2344.2 Volume:1148 ValidationIssues:} {Time:2024-08-05 07:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:45:00 +1000 AEST Open:2368.21 High:2368.32 Low:2368.21 Close:2368.32 Volume:481 ValidationIssues:} {Time:2024-08-05 07:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 07:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:10:00 +1000 AEST Open:2326.91 High:2334.13 Low:2326.91 Close:2334.13 Volume:129 ValidationIssues:} {Time:2024-08-05 08:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:25:00 +1000 AEST Open:2331.71 High:2338.45 Low:2331.71 Close:2338.45 Volume:130 ValidationIssues:} {Time:2024-08-05 08:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 08:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:45:00 +1000 AEST Open:2288.74 High:2288.74 Low:2279.86 Close:2279.86 Volume:502 ValidationIssues:} {Time:2024-08-05 09:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 09:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:15:00 +1000 AEST Open:2279 High:2279 Low:2279 Close:2279 Volume:49 ValidationIssues:} {Time:2024-08-05 10:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 10:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:55:00 +1000 AEST Open:2276.11 High:2276.11 Low:2276.11 Close:2276.11 Volume:18 ValidationIssues:} {Time:2024-08-05 11:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:25:00 +1000 AEST Open:2269.41 High:2272.05 Low:2269.41 Close:2272.05 Volume:2665 ValidationIssues:} {Time:2024-08-05 11:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:45:00 +1000 AEST Open:2265.56 High:2265.56 Low:2265.56 Close:2265.56 Volume:45 ValidationIssues:} {Time:2024-08-05 11:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 11:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 12:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:55:00 +1000 AEST Open:2224.64 High:2224.64 Low:2224.64 Close:2224.64 Volume:187 ValidationIssues:} {Time:2024-08-05 23:00:00 +1000 AEST Open:2227.71 High:2237.11 Low:2216.47 Close:2229.4 Volume:1397 ValidationIssues:} {Time:2024-08-05 23:05:00 +1000 AEST Open:2234.31 High:2249.11 Low:2224.75 Close:2234.07 Volume:1171 ValidationIssues:} {Time:2024-08-05 23:10:00 +1000 AEST Open:2245.91 High:2247.49 Low:2240.15 Close:2247.49 Volume:221 ValidationIssues:} {Time:2024-08-05 23:15:00 +1000 AEST Open:2218.39 High:2218.39 Low:2218.39 Close:2218.39 Volume:4 ValidationIssues:} {Time:2024-08-05 23:20:00 +1000 AEST Open:2218.2 High:2218.2 Low:2209.9 Close:2209.9 Volume:114 ValidationIssues:} {Time:2024-08-05 23:25:00 +1000 AEST Open:2194.61 High:2198.7 Low:2190.51 Close:2198.7 Volume:75 ValidationIssues:} {Time:2024-08-05 23:30:00 +1000 AEST Open:2248.77 High:2248.77 Low:2248.77 Close:2248.77 Volume:1 ValidationIssues:} {Time:2024-08-05 23:35:00 +1000 AEST Open:2304 High:2304 Low:2304 Close:2304 Volume:50 ValidationIssues:} {Time:2024-08-05 13:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 13:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 13:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 13:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:10:00 +1000 AEST Open:2348.51 High:2348.51 Low:2346.6 Close:2346.6 Volume:113 ValidationIssues:} {Time:2024-08-06 00:15:00 +1000 AEST Open:2375 High:2375 Low:2374.96 Close:2375 Volume:280 ValidationIssues:} {Time:2024-08-05 14:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:25:00 +1000 AEST Open:2375.46 High:2375.46 Low:2373.4 Close:2373.4 Volume:841 ValidationIssues:} {Time:2024-08-05 14:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:40:00 +1000 AEST Open:2410.2 High:2410.2 Low:2410.2 Close:2410.2 Volume:30 ValidationIssues:} {Time:2024-08-06 00:45:00 +1000 AEST Open:2417.4 High:2417.4 Low:2417.4 Close:2417.4 Volume:560 ValidationIssues:} {Time:2024-08-05 14:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 14:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:00:00 +1000 AEST Open:2380.2 High:2380.2 Low:2380.2 Close:2380.2 Volume:18 ValidationIssues:} {Time:2024-08-05 15:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 15:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:25:00 +1000 AEST Open:2496 High:2512.52 Low:2496 Close:2510.3 Volume:37 ValidationIssues:} {Time:2024-08-06 02:30:00 +1000 AEST Open:2512.52 High:2512.52 Low:2496.84 Close:2496.84 Volume:50 ValidationIssues:} {Time:2024-08-06 02:35:00 +1000 AEST Open:2491.08 High:2491.08 Low:2491.08 Close:2491.08 Volume:20 ValidationIssues:} {Time:2024-08-05 16:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 16:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:30:00 +1000 AEST Open:2466.6 High:2466.6 Low:2466.6 Close:2466.6 Volume:10 ValidationIssues:} {Time:2024-08-05 17:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 17:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:50:00 +1000 AEST Open:2479.2 High:2479.2 Low:2479.2 Close:2479.2 Volume:1 ValidationIssues:} {Time:2024-08-05 17:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:40:00 +1000 AEST Open:2399.89 High:2400.85 Low:2399.89 Close:2400.85 Volume:50 ValidationIssues:} {Time:2024-08-05 18:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 18:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:55:00 +1000 AEST Open:2396.53 High:2396.53 Low:2396.53 Close:2396.53 Volume:20 ValidationIssues:} {Time:2024-08-05 19:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:15:00 +1000 AEST Open:2383.09 High:2383.09 Low:2374.45 Close:2374.45 Volume:52 ValidationIssues:} {Time:2024-08-05 19:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 19:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:50:00 +1000 AEST Open:2376.11 High:2376.11 Low:2376.11 Close:2376.11 Volume:32 ValidationIssues:} {Time:2024-08-05 19:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 20:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:45:00 +1000 AEST Open:2437.44 High:2437.44 Low:2432.64 Close:2432.64 Volume:40 ValidationIssues:} {Time:2024-08-05 21:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 21:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 22:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-05 23:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:00:00 +1000 AEST Open:2421 High:2423.88 Low:2421 Close:2423.88 Volume:40 ValidationIssues:} {Time:2024-08-06 00:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:25:00 +1000 AEST Open:2509.77 High:2509.77 Low:2509 Close:2509 Volume:250 ValidationIssues:} {Time:2024-08-06 00:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:45:00 +1000 AEST Open:2529 High:2536.15 Low:2529 Close:2536.15 Volume:271 ValidationIssues:} {Time:2024-08-06 00:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 00:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:00:00 +1000 AEST Open:2544.24 High:2544.88 Low:2544.24 Close:2544.88 Volume:700 ValidationIssues:} {Time:2024-08-06 01:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 01:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:20:00 +1000 AEST Open:2519.8 High:2520.96 Low:2519.8 Close:2520.96 Volume:773 ValidationIssues:} {Time:2024-08-06 02:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 02:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:25:00 +1000 AEST Open:2502.12 High:2502.12 Low:2502.12 Close:2502.12 Volume:1 ValidationIssues:} {Time:2024-08-06 03:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 03:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:55:00 +1000 AEST Open:2494.92 High:2496.96 Low:2494.92 Close:2496.96 Volume:207 ValidationIssues:} {Time:2024-08-06 04:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 04:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 05:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:30:00 +1000 AEST Open:2523.44 High:2523.44 Low:2523.44 Close:2523.44 Volume:30 ValidationIssues:} {Time:2024-08-06 06:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:45:00 +1000 AEST Open:2521.96 High:2521.96 Low:2521.96 Close:2521.96 Volume:25 ValidationIssues:} {Time:2024-08-06 06:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 06:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:20:00 +1000 AEST Open:2508.36 High:2508.36 Low:2508.36 Close:2508.36 Volume:22 ValidationIssues:} {Time:2024-08-06 07:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 07:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:35:00 +1000 AEST Open:2468.52 High:2468.52 Low:2461.32 Close:2461.32 Volume:41 ValidationIssues:} {Time:2024-08-06 18:40:00 +1000 AEST Open:2461.32 High:2461.32 Low:2461.32 Close:2461.32 Volume:40 ValidationIssues:} {Time:2024-08-06 08:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 08:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:55:00 +1000 AEST Open:2474.28 High:2474.28 Low:2468.04 Close:2468.04 Volume:30 ValidationIssues:} {Time:2024-08-06 09:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 09:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 10:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:25:00 +1000 AEST Open:2442.72 High:2442.72 Low:2442 Close:2442 Volume:189 ValidationIssues:} {Time:2024-08-06 11:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 11:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:15:00 +1000 AEST Open:2463.24 High:2463.24 Low:2463.24 Close:2463.24 Volume:30 ValidationIssues:} {Time:2024-08-06 12:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 12:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:45:00 +1000 AEST Open:2431.08 High:2431.08 Low:2431.08 Close:2431.08 Volume:20 ValidationIssues:} {Time:2024-08-06 12:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:55:00 +1000 AEST Open:2435.52 High:2435.52 Low:2434.92 Close:2434.92 Volume:31 ValidationIssues:} {Time:2024-08-06 13:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:05:00 +1000 AEST Open:2446.44 High:2446.44 Low:2446.44 Close:2446.44 Volume:20 ValidationIssues:} {Time:2024-08-06 13:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:25:00 +1000 AEST Open:2424.47 High:2424.47 Low:2424.47 Close:2424.47 Volume:30 ValidationIssues:} {Time:2024-08-06 13:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:40:00 +1000 AEST Open:2431.32 High:2431.32 Low:2431.2 Close:2431.2 Volume:288 ValidationIssues:} {Time:2024-08-06 13:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 13:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 14:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 14:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:10:00 +1000 AEST Open:2520 High:2520 Low:2520 Close:2520 Volume:10 ValidationIssues:} {Time:2024-08-07 00:15:00 +1000 AEST Open:2531.22 High:2543.84 Low:2531.22 Close:2543.84 Volume:32 ValidationIssues:} {Time:2024-08-06 14:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:25:00 +1000 AEST Open:2539.67 High:2539.67 Low:2539.67 Close:2539.67 Volume:20 ValidationIssues:} {Time:2024-08-06 14:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:35:00 +1000 AEST Open:2536.44 High:2536.44 Low:2536.44 Close:2536.44 Volume:10 ValidationIssues:} {Time:2024-08-07 00:40:00 +1000 AEST Open:2532.52 High:2532.52 Low:2531.75 Close:2531.75 Volume:2665 ValidationIssues:} {Time:2024-08-07 00:45:00 +1000 AEST Open:2538.37 High:2538.37 Low:2533.05 Close:2533.05 Volume:6480 ValidationIssues:} {Time:2024-08-06 14:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 14:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 01:40:00 +1000 AEST Open:2517.72 High:2517.72 Low:2517.72 Close:2517.72 Volume:50 ValidationIssues:} {Time:2024-08-07 01:45:00 +1000 AEST Open:2510.42 High:2510.42 Low:2510.42 Close:2510.42 Volume:3 ValidationIssues:} {Time:2024-08-06 15:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 15:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 02:05:00 +1000 AEST Open:2523.05 High:2523.05 Low:2523.05 Close:2523.05 Volume:2 ValidationIssues:} {Time:2024-08-07 02:10:00 +1000 AEST Open:2529.68 High:2529.68 Low:2529.68 Close:2529.68 Volume:44 ValidationIssues:} {Time:2024-08-06 16:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 16:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 02:55:00 +1000 AEST Open:2537.72 High:2537.72 Low:2537.72 Close:2537.72 Volume:30 ValidationIssues:} {Time:2024-08-06 17:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 03:30:00 +1000 AEST Open:2514.97 High:2514.97 Low:2514.85 Close:2514.85 Volume:1026 ValidationIssues:} {Time:2024-08-06 17:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 03:45:00 +1000 AEST Open:2504.72 High:2504.72 Low:2504.72 Close:2504.72 Volume:30 ValidationIssues:} {Time:2024-08-06 17:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 17:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 04:05:00 +1000 AEST Open:2493.12 High:2493.12 Low:2493.12 Close:2493.12 Volume:1 ValidationIssues:} {Time:2024-08-06 18:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 18:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 05:00:00 +1000 AEST Open:2498.72 High:2499.37 Low:2497.42 Close:2498.07 Volume:181 ValidationIssues:} {Time:2024-08-06 19:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 05:40:00 +1000 AEST Open:2501.32 High:2501.32 Low:2501.32 Close:2501.32 Volume:3 ValidationIssues:} {Time:2024-08-06 19:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 19:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 06:00:00 +1000 AEST Open:2480.52 High:2484.96 Low:2480.52 Close:2484.96 Volume:186 ValidationIssues:} {Time:2024-08-06 20:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 06:15:00 +1000 AEST Open:2484.86 High:2491.2 Low:2484.86 Close:2491.2 Volume:518 ValidationIssues:} {Time:2024-08-06 20:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 20:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 21:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 22:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 08:25:00 +1000 AEST Open:2463.24 High:2464.32 Low:2463.24 Close:2464.32 Volume:1026 ValidationIssues:} {Time:2024-08-07 08:30:00 +1000 AEST Open:2475.84 High:2475.84 Low:2475.84 Close:2475.84 Volume:30 ValidationIssues:} {Time:2024-08-06 22:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 08:40:00 +1000 AEST Open:2471.88 High:2471.88 Low:2471.88 Close:2471.88 Volume:2 ValidationIssues:} {Time:2024-08-06 22:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 08:50:00 +1000 AEST Open:2475.72 High:2475.72 Low:2475.72 Close:2475.72 Volume:49 ValidationIssues:} {Time:2024-08-06 22:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:15:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 09:20:00 +1000 AEST Open:2473.8 High:2473.8 Low:2473.8 Close:2473.8 Volume:3 ValidationIssues:} {Time:2024-08-06 23:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:40:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:50:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-06 23:55:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:00:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:05:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:10:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 10:15:00 +1000 AEST Open:2449.8 High:2450.76 Low:2449.8 Close:2450.76 Volume:44 ValidationIssues:} {Time:2024-08-07 00:20:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:25:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:30:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 00:35:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:} {Time:2024-08-07 10:40:00 +1000 AEST Open:2461.8 High:2461.8 Low:2460.36 Close:2460.36 Volume:118 ValidationIssues:} {Time:2024-08-07 00:45:00 +0000 UTC Open:0 High:0 Low:0 Close:0 Volume:0 ValidationIssues:}]
```

It appears most candles do not get data. I would just like to know if its the endpoint, or a `asset.Futures` implementation issue

Reply:
Couldn't get why this was giving this erorr ... Can I get back to it tomorrow?

Reply:
The endpoint seems to have an issue. So, I will further test it in detail tomorrow

Reply:
> Can I get back to it tomorrow?

Of course, take whatever time you need 🕊️

Reply:
Alright, I've found the issue. Kucoin Futures Kline limit is 500, whereas Spot it 1500. There is the Features variable `GlobalResultLimit` set to `1500`. I suggest changing it to `500`, if we make a couple more API requests to get candle data, I think its better than having a broken futures response. Feel free to investigate a more thorough solution, but I think it works fine

Reply:
I am unsure what rules or logic that Kucoin is following for that futures endpoint. It has sporadic data. I even tried without the end time given their docs and it still misses seemingly random candles.
I think leave it. The work is there to collect the data if Kucoin ever chooses to actually supply data properly